docs/advanced-configuration.md

```diff
@@ -104,6 +104,55 @@ There are following known limitations of native OTel logs collection:
   Logs collection with fluentd is still recommended if chart deployed with `autodetect.istio=true`.
 - Journald logs cannot be collected natively by Splunk OTel Collector yet.
 
+### Add log files from Kubernetes host machines/volumes
+
+You can add additional log files to be ingested from Kubernetes host machines and kubernetes volumes by configuring `agent.extraVolumes`, `agent.extraVolumeMounts` and `logsCollection.extraFileLogs` in the values.yaml file used to deploy Splunk OpenTelemetry Collector for Kubernetes.
+
+Example of adding audit logs from Kubernetes host machines
+
+```yaml
+logsCollection:
+  extraFileLogs:
+    filelog/audit-log:
+      include: [/var/log/kubernetes/apiserver/audit.log]
+      start_at: beginning
+      include_file_path: true
+      include_file_name: false
+      resource:
+        com.splunk.source: /var/log/kubernetes/apiserver/audit.log
+        host.name: 'EXPR(env("K8S_NODE_NAME"))'
+        com.splunk.sourcetype: kube:apiserver-audit
+agent:
+  extraVolumeMounts:
+    - name: audit-log
+      mountPath: /var/log/kubernetes/apiserver
+  extraVolumes:
+    - name: audit-log
+      hostPath:
+        path: /var/log/kubernetes/apiserver
+```
+
+### Processing Multi-Line Logs
+
+Splunk OpenTelemetry Collector for Kubernetes supports parsing of multiline logs to help read, understand and troubleshoot the multiline logs in a better way.
```

Comment:
Splunk OpenTelemetry Collector for Kubernetes supports parsing of multi-line logs to help read, understand, and troubleshoot the multi-line logs in a better way.